src/pages/resources/reference/push-notifications/templates/tabs/catalog.md

```diff
@@ -0,0 +1,105 @@
+| Channel ID | ⛔️ | `adb_channel_id` | string | The [notification's channel id](https://developer.android.com/guide/topics/ui/notifiers/notifications#ManageChannels) (new in Android O). The app must create a channel with this channel ID before any notification with this channel ID is received. If you don't send this channel ID in the request, or if the channel ID provided has not yet been created by the app, FCM uses the channel ID specified in the app manifest.<br />If not provided in payload, SDK uses a "default" channel ID of value "CampaignPushChannel".<br />If < API 26 (Android O), this value is ignored. |
+| Badge Count | ⛔️ | `adb_n_count` | string | Value to be used on app's badge. |
+| Priority | ⛔️ | `adb_n_priority` | string | Designates the notification priority for Android - [importance](https://developer.android.com/reference/android/app/NotificationChannel#setImportance(int)) for API >= 26 (Android 8.0) or [priority](https://developer.android.com/reference/androidx/core/app/NotificationCompat.Builder#setPriority(int)) for API < 26 |
+| Tag | ⛔️ | `adb_tag` | string | Identifier used to replace existing notifications in the notification drawer. If not specified, each request creates a new notification. If specified and a notification with the same tag is already being shown, the new notification replaces the existing one in the notification drawer. |
+| Sticky | ⛔️ | `adb_sticky` | string | Boolean value represented as a string "true" or "false".<br />When set to false or unset, the notification is automatically dismissed when the user clicks it in the panel. When set to true, the notification persists even when the user clicks it. |
+| Ticker | ⛔️ | `adb_ticker` | string | Sets the "ticker" text, which is sent to accessibility services. Prior to API level 21 (Lollipop), sets the text that is displayed in the status bar when the notification first arrives. |
+| Color - Small Icon | ⛔️ | `adb_clr_icon` | string | Color for the notification's small icon.<br />Represented as six character hex, e.g. `00FF00` |
+| Color - Background | ⛔️ | `adb_clr_bg` | string | Color for notification's background.<br />Represented as six character hex, e.g. `00FF00` |
+| Call to action button text | ✅ | `adb_cta_txt` | string | Text to be shown on the CTA button |
+| Call to action button link | ✅ | `adb_cta_uri` | string | URI to be handled when the user clicks the CTA button |
+| Call to action button color | ✅ | `adb_cta_clr` | string | Color for the CTA button. Represented as six character hex, e.g. `00FF00` |
+| Call to action button text color | ✅ | `adb_cta_txt_clr` | string | Color for the CTA button text. Represented as six character hex, e.g. `00FF00` |
```

Comment:
are these actually required?  we don't have default values?

Reply:
there are default values (black text for light mode and white text for dark mode) but we decided to make the CTA text color customizable as the CTA button color is customizable. This will allow the developer to make the text have better contrast (for example when using a lighter button color in dark mode and the white text may become unreadable).